plugins/pencilblue/controllers/api/admin/site_settings/email/send_test.js

```diff
@@ -24,26 +24,34 @@ function SendTestEmail(){}
 //inheritance
 util.inherits(SendTestEmail, pb.FormController);
 
-SendTestEmail.prototype.onPostParamsRetrieved = function(post, cb) {
+SendTestEmail.prototype.render = function(cb) {
     var self = this;
 
-    var message = this.hasRequiredParams(post, this.getRequiredFields());
-    if(message) {
-        cb({content: pb.BaseController.apiResponse(pb.BaseController.API_FAILURE, message)});
-        return;
-    }
-
-    var options = {
-        to: post.email,
-        subject: 'Test email from PencilBlue',
-        layout: 'This is a successful test email from the PencilBlue system.',
-    };
-    pb.email.sendFromLayout(options, function(err, response) {
-        if(err) {
-            cb({content: pb.BaseController.apiResponse(pb.BaseController.API_FAILURE, JSON.stringify(err))});
+    this.getJSONPostParams(function(err, post) {
```

Comment:
Is this something that can be added to the getRoutes function in plugins too? I want to keep this controller in line with what plugins need, so users can use our work as a reference.